Automattic/wp-calypso
# Subscriptions: Paid subscribers content importer next iteration

Related to https://github.com/Automattic/wp-calypso/issues/93150

## Proposed Changes

It:
- automatically uploads the content file
- fixes some UI issues to make it closer to the designs

I know I copied a lot of files but I'll try to remove them if possible.

## Why are these changes being made?


To make content importer UI look closer to the designs.

## Testing Instructions



1. Go to `/import/newsletter/substack/{blogHostname}`
2. Type the Substack blog URL into the input and click "Continue"
3. Make sure the "Content" importer works but still needs a lot of changes to make it as on the designs

## Pre-merge Checklist
